libs/martech/attributes.js

```diff
@@ -19,7 +19,14 @@ export function decorateDefaultLinkAnalytics(block, config) {
     && block.nodeName === 'DIV') {
     let header = '';
     let linkCount = 1;
-    block.querySelectorAll('h1, h2, h3, h4, h5, h6, a:not(.video.link-block), button, .tracking-header').forEach((item) => {
+
+    let headerSelector = 'h1, h2, h3, h4, h5, h6, .tracking-header';
+    const headers = block.querySelectorAll(headerSelector);
+    if (!headers.length) headerSelector = `${headerSelector}, b, strong`;
+    block.querySelectorAll(`${headerSelector}, a:not(.video.link-block), button`).forEach((item) => {
+      if (item.nodeName === 'STRONG' || item.nodeName === 'B') {
```

Comment:
nit: it's a bit hard to see that if this condition is true then the else condition on line 53 is  what's next.  Maybe instead something like:
```
  if (item.nodeName === 'STRONG' || item.nodeName === 'B') {
    item.classList.add('tracking-header');
  }

  if (item.nodeName !== 'A' && item.nodeName !== 'BUTTON') {
    header = processTrackingLabels(item.textContent, config, 20);
    return;
  }

  if (item.classList.contains('tracking-header')) {
  ...
```

Reply:
@chrischrischris I think the issue is that the strong or bold check really only needs to happen if it's not a link or button. but if it's not, I want to add also add that class for the bold/strong tag.  So I moved the if.